modules/ROOT/pages/queries-aggregations/filtering.adoc

```diff
@@ -127,10 +164,9 @@ const features = {
 const neoSchema = new Neo4jGraphQL({ features, typeDefs, driver });
 ----
 
-> The nature of RegEx matching means that on an unprotected API, this could potentially be used to execute a ReDoS attack (https://owasp.org/www-community/attacks/Regular_expression_Denial_of_Service_-_ReDoS) against the backing Neo4j database.
+// This next section seems to be wrong. It mentions 4 operators, but there's one 2 and a similar sentence is repicated with not much context. Please take a look.
```

Comment:
@mjfwebb did you have the chance to check this? I merged the PR and just remembered I left this note